test/suites/smoke/test-dynamic-fees.ts

```diff
@@ -97,7 +100,15 @@ describeSuite({
 
     beforeAll(async function () {
       paraApi = context.polkadotJs("para");
-      const { specVersion, specName } = paraApi.consts.system.version;
+
+      const blockNumArray = atBlock > 0 ? [atBlock] : await getBlockArray(paraApi, timePeriod);
+      const previousBlockNumber = Math.max(blockNumArray[0] - 1, 0);
```

Comment:
Not sure, but maybe this is preferable:

```suggestion
      const previousBlockNumber = Math.max(blockNumArray[blockNumArray.length - 1] - 1, 0);
```

Reply:
No, we want the block that is before the first block of the array.
I added comment